### DISCUSSION TITLE
Subject: Inquiry Regarding ENS Acceptance and Denial in Viem Repository

### DISCUSSION THREAD

**Discussion by defiQUG** [wevm/viem#2343] [General]
Dear Viem Repository Administration,

I hope this message finds you well. I am reaching out to initiate a discussion regarding the acceptance and denial of Ethereum Name Service (ENS) addresses within the Viem repository on GitHub.com. I have noticed that there have been discussions surrounding this topic, and I would greatly appreciate your guidance and clarification on the matter.

Specifically, I am seeking information on what qualifies as an "official ENS" according to the Viem repository. Furthermore, I am confused due to the recent partnership between ENS.domains and GoDaddy.com, which allows for the seamless import of domains from GoDaddy.com into the ENS domain system.

Given this partnership, whereby domains can be easily migrated from GoDaddy.com to ENS.domains, I am unsure of how Viem's acceptance and denial policies regarding ENS addresses are affected. It would be immensely helpful if you could provide some insights into this matter and elaborate on the reasoning behind any decisions made.

Understanding your perspective and the policies surrounding ENS addresses within the Viem repository would not only clear up my confusion but also enable me to align my project integration plans accordingly.

I appreciate your attention and time dedicated to addressing this inquiry. I am eagerly looking forward to your response and the opportunity to engage in this discussion.

Thank you and best regards,

defiQUG
GitHub.com User

Reference: https://aboutus.godaddy.net/newsroom/company-news/news-details/2024/GoDaddy-and-Ethereum-Name-Service-Bridge-the-Gap-Between-Domain-Names-and-Crypto-Wallets/default.aspx

**Comment by jxom** [comment]
I am unsure what you are trying to ask.

If you are asking what constitutes as an official ENS contract address, it is the contracts that are deployed by ENS and listed here: https://docs.ens.domains/learn/deployments

**Reply by defiQUG** [reply]
I don't understand why the ENS components were rejected for the following:

https://app.ens.domains/defi-oracle.io

On Etherscan, this is the transaction record: https://etherscan.io/name-lookup-search?id=defi-oracle.io

Overview of ENS
Resolved Address:  Address [0x34eCe27C724291C7233dC8114f76C1169f0863C8 ](https://etherscan.io/address/0x34eCe27C724291C7233dC8114f76C1169f0863C8)
Controller:                               [0x34eCe27C724291C7233dC8114f76C1169f0863C8 ](https://etherscan.io/address/0x34eCe27C724291C7233dC8114f76C1169f0863C8) 
DNS Owner:                            [0x34eCe27C724291C7233dC8114f76C1169f0863C8 ](https://etherscan.io/address/0x34eCe27C724291C7233dC8114f76C1169f0863C8) 
Token ID:        11430619665414158364559710630959690439401577322786590663145894306013562812376
